kernel/packages/entryPoints/editor.ts

```diff
@@ -22,7 +22,7 @@ import {
 import defaultLogger from '../shared/logger'
 import { uuid } from '../decentraland-ecs/src/ecs/helpers'
 import { Vector3 } from '../decentraland-ecs/src/decentraland/math'
-import { sceneLifeCycleObservable } from '../decentraland-loader/lifecycle/controllers/scene'
+import { sceneLifeCycleObservable } from 'shared/sceneLifeCycleObservable'
```

Comment:
this scares me! there shouldn't be a reference to `decentraland-loader` (that probably was making the executable a lot bigger than it was needed)